GoogleForCreators/web-stories-wp
# Video Trimming: Enable by default

## Context

As this feature has been both bug-bashed and thoroughly QA'ed through many iterations, this removes the feature flag in time for release.

A full test suite is still remaining, will be implemented in a separate PR.

## User-facing changes

_Video trim is enabled_

## Testing Instructions

This PR can be tested by following these steps:

1. Observe that the video trim experiment is enabled by default on the experiments page
2. Create a new story and add a video
3. Observe the video trim mode still exists and still works

## Checklist




---

See #8893
